pkg/controllers/cassandra/cassandra_test.go

```diff
@@ -1,30 +1,22 @@
-package cassandra
+package cassandra_test
```

Comment:
I guess you've done this so you can't access non-exported fields from the `cassandra` package during tests? If so, okay 😄

Reply:
In the previous version of the tests I mistakenly thought that I needed to wait for the cache sync before simulating resource  create / update / delete  in the tests.
So I grabbed the private cassListerSynced attribute from the controller.
But experimented some more and found that that isn't actually necessary.